render/json/default.py

```diff
@@ -223,6 +223,27 @@ def renderEntry(self, skel, actionName, params = None):
 		request.current.get().response.headers["Content-Type"] = "application/json"
 		return json.dumps(res)
 
+	def preprocessParams(self, params):
```

Comment:
make it private, rename to "_preprocessParams" ... IMHO, this piece of code must not be put into a separate function.

Reply:
I used an extra method for this to make it easy possible to modify this part and do other magic with the params.

Reply:
you are right! :+1: